package.json

```
@@ -7,7 +7,8 @@
     "run-testcases": "node lib/checkAbnf.js",
     "test": "npm run generate-grammar && node lib/checkAbnf.js",
     "watch": "pwsh lib/check-abnf.ps1 -watch",
-    "tool-test": "c8 mocha"
+    "tool-test": "c8 mocha",
+    "railroad": "abnf-to-railroad abnf/odata-abnf-construction-rules.txt syntax-diagram/odata-abnf-diagrams.html"
```

```suggestion
    "railroad": "npx abnf-to-railroad abnf/odata-abnf-construction-rules.txt syntax-diagram/odata-abnf-diagrams.html"
```
Use the tool version installed in the package, not the globally installed one.

---

Ah, [not needed](https://docs.npmjs.com/cli/v8/commands/npm-run-script):

> In addition to the shell's pre-existing `PATH`, `npm run` adds `node_modules/.bin` to the `PATH` provided to scripts.